scripts/jenkins-yoctobuild-build.sh

```diff
@@ -182,12 +182,11 @@ if [ "$RUN_INTEGRATION_TESTS" = "true" ]; then
     docker build -t mendersoftware/mender-client-qemu:latest --build-arg VEXPRESS_IMAGE=core-image-full-cmdline-vexpress-qemu.sdimg --build-arg UBOOT_ELF=u-boot.elf .
     cd $WORKSPACE/integration/tests && ./run.sh
 
-    if [ $PUBLISH_ARTIFACTS = "true" ]; then
-        s3cmd -F put core-image-full-cmdline-vexpress-qemu.ext4 s3://mender/temp/core-image-full-cmdline-vexpress-qemu.ext4
+    if [ $PUBLISH_ARTIFACTS = "true" ] && [ ! -z "$RELEASE_VERSION" ]; then
+        s3cmd -F put core-image-full-cmdline-vexpress-qemu.ext4 s3://mender/temp_${RELEASE_VERSION}/core-image-full-cmdline-vexpress-qemu.ext4
         s3cmd setacl s3://mender/temp/core-image-full-cmdline-vexpress-qemu.ext4 --acl-public
 
-        if [ ! -z "${RELEASE_VERSION}" ]; then
-
+        if [ ! -z "${RELEASE_VERSION}" ] && [ $PERFORM_RELEASE = "true" ]; then
```

Comment:
You can remove the check for `RELEASE_VERSION` entirely here, at this point you already know it's set.